Telegram/SourceFiles/history/history_message.cpp

```diff
@@ -463,7 +463,7 @@ void HistoryMessageReply::updateName() const {
 			w += st::msgServiceFont->spacew + _replyToVia->_maxWidth;
 		}
 
-		_maxReplyWidth = previewSkip + qMax(w, qMin(replyToText.maxWidth(), qint32(st::maxSignatureSize)));
+		_maxReplyWidth = previewSkip + std::max(w, std::min(replyToText.maxWidth(), qint32(st::maxSignatureSize)));
```

Comment:
clamp

Reply:
Clamp cant be used here. There may be situations it will lead to undefined behavior, this constuction doesnt guarantee that second argument of std:max greater 0 and first argument of std::min greater 0, or one of these greater than other

From specs of std::clamp
"The behavior is undefined if the value of lo is greater than hi"